Neos.Flow/Migrations/Mysql/Version20180622074421.php

```diff
@@ -1,6 +1,7 @@
 <?php
 namespace Neos\Flow\Persistence\Doctrine\Migrations;
 
+use Doctrine\DBAL\Platforms\MySQLPlatform;
```

Comment:
Nice, but not used futher down, this still uses `$this->connection->getDatabasePlatform()->getName() != 'mysql'`